bigbluebutton-html5/imports/ui/components/dropdown/styles.scss

```diff
@@ -205,7 +205,7 @@
 
   &:after,
   &:before {
-    left: calc(var(--dropdown-caret-width) / 2);
+    left: calc(var(--dropdown-caret-width));
```

Comment:
Line 197 for *right* caret is unchanged. Should we not keep them consistent?

Reply:
If we change Line 197 to match this, the options dropdown gets misaligned.

![image](https://user-images.githubusercontent.com/22058534/48921471-01deec00-ee6e-11e8-84e5-4195da38c923.png)